02week/bio/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <meta charset="UTF-8">
+    <title>title</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+
+
+<div class="header">
+  <header> DEVELOPER. URBANIST. WRITER. </header>
+</div>
+  <div class="aboutme">
+  <!-- <p> I'm a web developer, writer, and game-changer in Austin, TX.
+    Click below to get to know me. </p> -->
+    <a href='file:///Users/emmapatton/Desktop/web-workbook/02week/bio/index.html'> HOME </a>
```

Comment:
Hey Emma! This may work while you are using your live server on atom, but it wont work for any other computer. This is basically looking through your personal/local files (hence the Users/emmapatton...). Instead, look for it just within the web-workbook folder. The href should look more like this: <a href="./index.html"> (may be slightly off, I'm not actually testing it)

Reply:
Okay it was hiding my stuff, do this: href="./index.html"